db/schema.rb

```diff
@@ -336,9 +336,9 @@
   create_table "mutes", force: :cascade do |t|
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.boolean "hide_notifications", default: true, null: false
```

Comment:
Not sure what this is about, the version string update is OK but this seems unnecessary, you can manually switch it around.

Reply:
Oops, this is the result of a bad merge. My bad for not seeing it!

Reverted